libcrux-ml-dsa/src/simd/avx2/encoding/gamma1.rs

```diff
@@ -0,0 +1,89 @@
+use libcrux_intrinsics::avx2::*;
+
+#[inline(always)]
+fn serialize_when_gamma1_is_2_pow_17<const OUTPUT_SIZE: usize>(
```

Comment:
Why replicate the code from portable here? Wouldn't it be better to convert to an array of integers and call the portable serialize?